format-all.el

```diff
@@ -1107,6 +1107,13 @@ Consult the existing formatters for examples of BODY."
   (:features)
   (:format (format-all--buffer-easy executable "fmt" "-")))
 
+(define-format-all-formatter meson-fmt
+  (:executable "meson")
+  (:install  "pip install meson")
+  (:languages "Meson")
+  (:features)
+  (:format (format-all--buffer-easy executable "fmt" "--editor-config" "-")))
```

Comment:
`fmt` is fine, but `format` seems to be the canonical name you can easily search for. ([Google](https://www.google.com/search?q=%22meson+fmt%22) did not give good results 🥲)
https://mesonbuild.com/Commands.html#format